Classes/ViewHelpers/Widget/LinkViewHelper.php

```diff
@@ -67,13 +67,10 @@ public function initializeArguments()
      */
     public function render($pluginName, $extensionName, $action = null, $arguments = array(), $section = '', $format = '', $ajax = true)
     {
-        if ($ajax === true) {
-            $uri = $this->getAjaxUri();
-        } else {
-            $uri = $this->getWidgetUri();
-        }
+        $uri = $ajax ? $this->getAjaxUri() : $this->getWidgetUri();
```

Comment:
Ah, okay. Thanks for the clarification.

So Fluid does not ensure the corresponding type?